control/mpc_lateral_controller/param/lateral_controller_defaults.param.yaml

```diff
@@ -47,7 +47,10 @@
     vehicle_model_type: "kinematics" # vehicle model type for mpc prediction. option is kinematics, kinematics_no_delay, and dynamics
     input_delay: 0.24 # steering input delay time for delay compensation
     vehicle_model_steer_tau: 0.3 # steering dynamics time constant (1d approximation) [s]
-    steer_rate_lim_dps: 600.0 # steering angle rate limit [deg/s]
+    steer_rate_lim_dps_list_by_curvature: [10.0, 20.0]            # steering angle rate limit list depending on curvature [deg/s]
+    curvature_list_for_steer_rate_lim: [0.001, 0.01]              # curvature list for steering angle rate limit interpolation [/m]
+    steer_rate_lim_dps_list_by_velocity: [10.0, 20.0]             # steering angle rate limit list depending on velocity [deg/s]
```

Comment:
Should `steer_rate_lim_dps_list` be in the opposite order?  It should be `[20.0, 10.0]`, not `[10.0, 20.0]`